brave/brave-core
# Implement window.ethereum.enable

Resolves https://github.com/brave/brave-browser/issues/17043

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Enable brave native wallet feature
2. Setup wallet & unlock it if needed
3. Visit https://permission.site and open dev tool
4. Call window.ethereum.enable() in dev tool should show the prompt
5. Accept the request from the prompt
6. Visit brave://settings/content/ethereum, should see entry for the site+address granted
